canonical-websites/www.ubuntu.com
# Create the AI/ML german takeover

## Done
Build the AI/ML webinar takeover in German
Fixed the existing takeovers to work in the new system.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Set you browser lang to German
- Refresh the homepage until you see the German AI takeover
- Check it matches [the copy doc](https://docs.google.com/document/d/1CQNGRpldiJ1js5fRczKc8kDDZOlrVyNkfYIJ3r8OCVc/edit)

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4809
